diffblue-base.json

```diff
@@ -1,5 +1,7 @@
 {
   "phaseBase": {
-    "javaVersion": 6
+    "javaVersion": 6,
+    "preferDepsJar": true,
+    "context-include": "com.jsoniter"
   }
 }
```

Comment:
:pick: Missing extra line

Reply:
You need to context-include `java.` too.

Reply:
@svorenova I don't understand what you mean
@peterschrammel Thanks -  I didn't realise that. How do I separate values? Or do I give the flag multiple times?

Reply:
`"context-include": [ "com.jsoniter.", "java.", "org.cprover." ]`